### DISCUSSION TITLE
如何设置docker的访问？

### DISCUSSION THREAD

**Discussion by ZER-ONE** [natfrp/wiki#447] [使用问题]
我在nas用frp穿透可以访问我的nas，但是无法访问里面的docker容器，像qbittorrent，jellyfin等，我设置了另外的隧道访问nas的特定端口，但是并不起作用。所以这总需求要怎么实现呢？或者用什么技术实现？

**Comment by yesugao** [comment]
QB - WEB UI设置 -   把 启用 Host header 属性验证 (关掉)
